index.md

```diff
@@ -3,8 +3,36 @@ layout: default
 title: Home
 nav_order: 1
 ---
-# Welcome!
+# Overview
 
-In here you can find the Documentation for the Fyde Enterprise Console.
+Welcome to Fyde! Fyde brings Zero Trust/BeyondCorp Security to the endpoint. In short, with Fyde, you can implement secure access to enterprise resources, whether they are on-prem or in the cloud, with a quick and easy configuration process.
 
-Please check the topics on the left, or use the search above to start.
+Fyde has three main components: an agent (Fyde App), a proxy (Fyde Proxy), and an administration console (Fyde Console). It is very simple to start using Fyde: register a proxy and invite your employees to install Fyde App. Check our [Quick Start]({{ site.baseurl }}{% link quickstart.md %})) to know more details.
+
+## About Zero Trust
+
+The days of the VPN are gone: it is no longer feasible to establish a secure network perimeter with a hybrid setup with cloud resources and the myriad of devices that need to access company resources from anywhere. One single breach on a VPN setup can be catastrophic depending on the network setup and configuration.
+
+Zero Trust builds upon the assertion that the network is assumed to be hostile. In consequence, network locality is not sufficient for establishing trust and every flow must be authenticated and authorized in a dynamic fashion. This creates an effective separation between the _control plane_ -- the supporting system that implements the flow authentication and authorization according to the defined policies -- and the _data plane_.
+
+To learn more about Zero Trust, check [Zero Trust Networks: Building Secure Systems in Untrusted Networks](https://www.amazon.com/Zero-Trust-Networks-Building-Untrusted/dp/1491962194) and the [BeyondCorp paper by Google](https://storage.googleapis.com/pub-tools-public-publication-data/pdf/43231.pdf)
+
+## Architecture overview
+
+As mentioned above, the Fyde architecture relies on three main components: an agent (Fyde App), a proxy (Fyde Proxy), and an administration console (Fyde Console). The Fyde agent operates at the network layer. When a device starts a connection to a protected resource, the Fyde agent intercepts it and opens a mTLS connection with the Fyde Proxy, sending also the device and user attributes. The Fyde Proxy will check the attributes and authorize or deny the connection to the resource. To configure policies, you can use the Fyde Console.
+
+![Architecture]({{ site.baseurl }}{% link imgs/fyde-architecture.png %})
```

Comment:
@GMartinez-Sisti is this ok?

Reply:
@luisalima To match the correct terminology we should use `Fyde Access Proxy` and `Fyde Enterprise Console`. The process looks alright for me.

Reply:
@GMartinez-Sisti I know, it's too large for the image 😭

Reply:
In the text!

Reply:
Ooops :D You're right, fixed 👍 Thanks!